environments/reana-demo-bsm-search/Dockerfile

```diff
@@ -1,15 +1,30 @@
 # Start from the ROOT6 base image:
 FROM reanahub/reana-env-root6
 
-# Install HFtools:
+# # Install HFtools:
+
+# Update and install dependendies
 RUN apt-get -y update && \
-    apt-get -y install \
-       python-pip \
-       zip && \
     apt-get autoremove -y && \
-    apt-get clean -y
-RUN pip install hftools
+    apt-get clean -y && \
+    apt-get install wget -y
+
+# Install a higher python version needed for hftools
+RUN version=3.6.8 && \
```

Comment:
It may look like an overkill to install Python from sources as part of the analysis container... I think two solutions would be simpler:

- either rely on installing some `hftools==0.0.4` or older version that was happy with Python 2;
- or update `reana-env-root6` environment to be based on Python3.  (Which we shall have to do anyway soon since Python 2 will be deprecated.)

The former is simpler to implement, and since HepData export isn't the focus of the BSM workflow, it can do until we implement the latter.